server/sonar-server/src/test/java/org/sonar/server/qualityprofile/QProfileFactoryMediumTest.java

```diff
@@ -208,7 +220,7 @@ public void fail_renaming_if_name_already_exists() {
   @Test
   public void delete() {
     initRules();
-    db.qualityProfileDao().insert(dbSession, QProfileTesting.newXooP1("org-123"));
+    db.qualityProfileDao().insert(dbSession, QProfileTesting.newXooP1(organization.getUuid()));
```

Comment:
I agree, but it is a lot of refactoring to change that. I deprecated the String parameter methods and created DTO parameter methods.